action.yml

```diff
@@ -16,6 +16,12 @@ inputs:
     description: '(Optional) Transforms environment variable name. Options: uppercase, lowercase, none. Default value: uppercase.'
     required: false
     default: 'uppercase'
+  auto-select-family-attempt-timeout:
+    description: '(Optional) Sets the default value of the autoSelectFamilyAttemptTimeout option. Changing this value would change the length of timeout between each call to get a secret from  a select family. Use this option when some calls are taking longer than the default timeout'
+    required: false
+    type: number
+    default: 1000
+    minimum: 0
```

Comment:
According to the [doc](https://nodejs.org/api/net.html#netsetdefaultautoselectfamilyattempttimeoutvalue) the minimum value is 10.